ShoppingCartApplication_CleanCodePractices/Cart.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ShoppingCartApplication_CleanCodePractices
+{
+    //Shopping Cart Application
+    public class Cart 
+    {
+        public List<CartItem> cartItemList = new List<CartItem>();
```

Comment:
Use proper access specifiers for properties

Reply:
Made setter's access to the List private, is that okay?